src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -890,6 +896,48 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		push(topBottomContainerViewController)
 	}
 
+	// MARK: - Test Type Selection
+	
+	// to.do 14717: Remove default value."
+	/// Shows the Type of Tests screen
+	/// - Parameter preSelectSelfTest: If the process was started via self-report with self-test on the Manage Your Tests View, the Self-test entry shall be pre-selected. Otherwise, no entry shall be selected.
+	private func showTestTypeSelectionScreen(preSelectSelfTest: Bool = false) {
+		let vc = TestTypeSelectionViewController(
+			viewModel: TestTypeSelectionViewModel(preSelectSelfTest: preSelectSelfTest),
+			onPrimaryButtonTap: { [weak self] submissionType in
+				self?.model.submissionTypeSRSSelected(submissionType)
+				self?.showNextScreenFromTestTypeSelectionScreen()
+			}, onDismiss: { [weak self] in
+				self?.parentViewController?.dismiss(animated: true)
+			}
+		)
+		
+		let footerViewController = FooterViewController(
+			FooterViewModel(
+				primaryButtonName: "Weiter",
+				primaryIdentifier: "todo",
+				isSecondaryButtonEnabled: false,
+				isSecondaryButtonHidden: true
+			)
+		)
+		
+		let topBottomContainerViewController = TopBottomContainerViewController(
+			topController: vc,
+			bottomController: footerViewController
+		)
+
+		present(topBottomContainerViewController)
+	}
+	
+	private func showNextScreenFromTestTypeSelectionScreen() {
```

Comment:
```suggestion
	private func showSRSFlowNextScreen() {
```
```suggestion
	private func showNextScreenFromTestTypeSelectionScreen() {
```

Reply:
mmh taste, I've changed it to your suggestion.